pkg/apiserver/index.go

```diff
@@ -26,10 +26,6 @@ import (
 // handleIndex is the root index page for Kubernetes.
 func handleIndex(req *restful.Request, resp *restful.Response) {
 	// TODO: use restful's Request/Response methods
-	if req.Request.URL.Path != "/" && req.Request.URL.Path != "/index.html" {
```

Comment:
There is not healthz on my localhost.
Have kept the "Welcome to Kubernetes" message for now.
Can change is later if required.